Dockerfile

```diff
@@ -30,7 +30,7 @@ COPY rel rel
 RUN mix release
 
 # prepare release image
-FROM alpine:3.17.3 AS app
+FROM alpine:3.20.3 AS app
 RUN apk add --no-cache --update bash openssl libstdc++
```

Comment:
Should we also install `ca-certificates` (for the new `public_key.cacerts_get`) and `ncurses` (I never learned why it is needed in Erlang, but I think my apps fail to start when it's missing)?